cluster/options.go

```diff
@@ -4,8 +4,16 @@ import "crypto/tls"
 
 // Options is exported
 type Options struct {
-	TLSConfig       *tls.Config
+	TLS             *TLSConfig
 	OvercommitRatio float64
 	Discovery       string
 	Heartbeat       uint64
 }
+
+// TLSConfig is exported
+type TLSConfig struct {
```

Comment:
the etcd go client does not use that `tls.Config` unfortunately and builds its own given cert files :-(